builder/image/templates/builder

```diff
@@ -112,8 +112,10 @@ if [ ! -f Dockerfile ]; then
     BUILD_OPTS+=' -v '
     BUILD_OPTS+=$(echo $CACHE_DIR)
     BUILD_OPTS+=':/tmp/cache:rw'
-    # give non-root slugbuilder user R/W perms for docker volumes
-    chown -R 2000:2000 $TMP_DIR $CACHE_DIR
+    # give slug group ownership of TMP_DIR and CACHE_DIR. This is used by the
+    # slug builder. Contents in these folders will be owned by slug:slug.
+    chown :2000 $TMP_DIR $CACHE_DIR
+    chmod g+rwx $TMP_DIR $CACHE_DIR
```

Comment:
We're not allowed to do that without root. 
See: https://superuser.com/questions/697608/chown-operation-not-permitted